.azure-pipelines.yml

```diff
@@ -0,0 +1,57 @@
+variables:
+  CONDASOURCE: "~/miniconda3/etc/profile.d/conda.sh"
```

Comment:
Might want to make a user with poor permissions.

Reply:
Agreed, its just using my local user on my laptop for testing. But its worth noting so I don't forget